src/Network.cpp

```diff
@@ -378,12 +643,19 @@ void Network::forward_cpu(std::vector<float>& input,
     // Input convolution
     constexpr int width = 19;
     constexpr int height = 19;
+    constexpr int tiles = (width + 1) * (height + 1) / 4;
     // Calculate output channels
     const auto output_channels = conv_biases[0].size();
+    //Assumes that residual blocks are identical and have same
+    //number of inputs and outputs
```

Comment:
I think that's a requirement for it to work as you need to add the input to the second layer.